src/common.mak

```diff
@@ -0,0 +1,204 @@
+# Common Makefile variables (shared between DMD, DRuntime and Phobos)
+# Be careful when updating
+# This file expects the following variables to be defined in the including Makefile:
+# - D_HOME        Root of all D repositories
+
+###############################################################################
+# Common directories
+################################################################################
+
+DMD_DIR = $(D_HOME)/dmd
+DRUNTIME_DIR = $(D_HOME)/druntime
+PHOBOS_DIR = $(D_HOME)/phobos
+DLANG_ORG_DIR = $(D_HOME)/dlang.org
+TOOLS_DIR = $(D_HOME)/tools
+INSTALL_DIR = $(D_HOME)/install
+INSTALLER_DIR = $(D_HOME)/installer
+TMP?=/tmp
+GIT_HOME=https://github.com/dlang
+
+# get OS and MODEL
+include $(D_HOME)/dmd/src/osmodel.mak
```

Comment:
Prolly we won't need this extra file now? I.e. we can inline its code here.

Reply:
It's still imported by all repos atm. We can inline it once all repositories are updated and depend on `common.mak`